SQLQuery_2509.sql

```diff
@@ -0,0 +1,57 @@
+--SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE='BASE TABLE';
+
+--CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+CREATE TABLE Employee(EmployeeID int primary key, FirstName varchar(20), LastName varchar(20),Department varchar(5),Salary int);
+
+--INSERT: Adds three employee records to the table.
+INSERT INTO Employee VALUES
+(1,'Samuel','Sam','CSE',20000),
+(2,'Karan','Kumar','ECE',25000),
+(3,'John','Doe','HR',40000);
+
+--SELECT: Shows different ways to query data:
+--Select all columns and rows
+SELECT * FROM employee;
+
+--Select specific columns
+SELECT FirstName,LastName FROM employee;
+SELECT FirstName,Department FROM employee;
+SELECT EmployeeID,LastName,Salary FROM employee;
+
+--Select with a WHERE clause to filter results
+SELECT FirstName,Department,Salary FROM employee WHERE(Salary > 22000);
```

Comment:
bracket is not necessary for single where filter

Reply:
Noted.